assembly/assembly-wsmaster-war/src/main/java/org/eclipse/che/api/deploy/WsMasterModule.java

```diff
@@ -409,6 +410,10 @@ private void configureMultiUserMode(
     bind(PermissionChecker.class).to(PermissionCheckerImpl.class);
 
     bindConstant().annotatedWith(Names.named("che.agents.auth_enabled")).to(true);
+
+    if (OpenShiftInfrastructure.NAME.equals(infrastructure)) {
+      install(new InfraProxyModule());
```

Comment:
can you move this line to https://github.com/eclipse/che/blob/74e4f757166d30a19dd296dfa601cdd0f30d05b0/assembly/assembly-wsmaster-war/src/main/java/org/eclipse/che/api/deploy/WsMasterModule.java#L263 ?